frontend/app/src/components/StreamlitDialog/SettingsDialog.tsx

```diff
@@ -178,13 +178,11 @@ export class SettingsDialog extends PureComponent<Props, UserSettings> {
   }
 
   private handleThemeChange = (index: number | null): void => {
-    const { activeTheme: oldTheme, availableThemes }: LibContextProps =
-      this.context
+    const { availableThemes }: LibContextProps = this.context
     const newTheme = availableThemes[index ?? 0]
 
-    this.props.metricsMgr.enqueue("themeChanged", {
-      oldThemeName: oldTheme.name,
-      newThemeName: newTheme.name,
+    this.props.metricsMgr.enqueue("menuClick", {
+      label: "changeTheme",
     })
```

Comment:
Do we not want to include the `oldThemeName` and `newThemeName`?

Reply:
The reasoning here is that the transition information itself doesn't provide very useful insights. IMO the main insight comes from the distribution of theme usage - also covered by the page profile - and the info that users are changing theme via frontend - covered by this menuClick event.

And loosely agreed schema for `menuClick` is only the `label` property, adding additional info here would require a bit more consideration with the data team.